lib/autoupdate/agent/updater.go

```diff
@@ -683,13 +700,13 @@ func (u *Updater) update(ctx context.Context, cfg *UpdateConfig, targetVersion s
 		if ok := revertConfig(ctx); ok {
 			u.Log.WarnContext(ctx, "Teleport updater encountered a configuration error and successfully reverted the installation.")
 		}
-		return trace.Errorf("failed to validate configuration for new version %q of Teleport: %w", targetVersion, err)
+		return trace.Errorf("failed to validate configuration for new version %q of Teleport: %w", target, err)
```

Comment:
`%q` always renders awkwardly with our logging config. Consider whether it's really necessary here. If we don't expect `target` to ever be the empty string then we can probably get by without quoting it in the error message.